WexHomework3/WexHomework3/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿static void IntializeArray(int[] array,int cofficient)
+{
+    for (int i = 0; i < array.Length; i++)
+    {
+        array[i] = i;
+
+        int k = 1;
+        while (k<cofficient)
```

Comment:
I think we can simplify this.
Please use [this ](https://docs.microsoft.com/en-us/dotnet/api/system.math.pow?view=net-6.0)